### PR TITLE
:ambulance: Incorrect popup open to indicate number of guests

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

File: pos_product_category_discount/doc/changelog.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 Updates
 =======
 
+`1.2.6`
+-------
+
+- **FIX:** Incorrect popup open to indicate number of guests
```
Comment (yelizariev) [inline pos_product_category_discount/doc/changelog.rst]: When?

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_product_category_discount/doc/changelog.rst (hunk 1 above)
Comment (GabbasovDinar) [reply pos_product_category_discount/doc/changelog.rst]: DONE

**Review by yelizariev — APPROVED** [review]

**Comment by yelizariev** [comment]
There are errors in tests

**Comment by yelizariev** [comment]
Still doesn't work

**Review by yelizariev — APPROVED** [review]

**Pull request merged**